bears/php/PHPCodeSnifferBear.py

```diff
@@ -0,0 +1,178 @@
+from coalib.bearlib.abstractions.Linter import linter
+from coalib.bearlib.spacing.SpacingHelper import SpacingHelper
+from coalib.bears.requirements.DistributionRequirement import (
+    DistributionRequirement)
+
+
+@linter(executable='phpcs',
+        output_format='regex',
+        config_suffix=".xml",
+        output_regex=r'(?P<filename>.+):(?P<line>\d+):(?P<column>\d+): '
+                      '(\w+) - (?P<message>.+)')
```

Comment:
you need to prepend `r` here too, otherwise python concats raw string and a normal string :+1: